app/uk/gov/hmrc/gform/sharedmodel/formtemplate/AuthConfig.scala

```diff
@@ -21,9 +21,9 @@ import uk.gov.hmrc.gform.sharedmodel.ValueClassFormat
 
 case class AuthConfig(
   authModule: AuthConfigModule,
-  predicates: Option[List[Predicate]],
   regimeId: RegimeId,
-  serviceId: Option[ServiceId]
+  serviceId: Option[ServiceId],
```

Comment:
Let's encode it into algebraic data structures as it was done in `SectionValidator` end get rid of `Option` and later `Option.get`.